### PR TITLE
Add some Unix console tests

### PR DESCRIPTION
**Pull request by hughbe**
Fixes some obvious coverage gaps running coverage on unix

### CODE REVIEW

File: src/System.Console/tests/WindowAndCursorProps.cs [hunk 1]
```diff
@@ -13,68 +13,145 @@ public class WindowAndCursorProps : RemoteExecutorTestBase
...
         Assert.Equal(Console.WindowHeight, Console.BufferHeight);
     }
 
     [Fact]
-    [PlatformSpecific(TestPlatforms.Windows)]  // Expected behavior specific to Windows
-    public static void WindowWidth_WindowHeight_InvalidSize()
+    [PlatformSpecific(TestPlatforms.AnyUnix)]  // Expected behavior specific to Unix
+    public static void BufferHeight_SetUnix_ThrowsPlatformNotSupportedException()
+    {
+        Assert.Throws<PlatformNotSupportedException>(() => Console.BufferWidth = 1);
```
Comment (danmosemsft) [inline src/System.Console/tests/WindowAndCursorProps.cs]: This duplicates BufferWidth_SetUnix_ThrowsPlatformNotSupportedException etc above?

File: src/System.Console/tests/WindowAndCursorProps.cs [hunk 2]
```diff
@@ -321,9 +412,77 @@ public static void GetCursorPosition()
         }
     }
 
+    [Fact]
+    public void CursorLeft_Set_GetReturnsExpected()
+    {
+        if (!Console.IsInputRedirected && !Console.IsOutputRedirected)
```
Comment (danmosemsft) [inline src/System.Console/tests/WindowAndCursorProps.cs]: When running the tests normally, does this block execute?

Comment (hughbe) [reply src/System.Console/tests/WindowAndCursorProps.cs]: Not sure - don't think so but this follows the other tests in this library, e.g.
```cs

    [Fact]
    [OuterLoop] // clears the screen, not very inner-loop friendly
    public static void Clear_Invoke_Success()
    {
        if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows) || (!Console.IsInputRedirected && !Console.IsOutputRedirected))
        {
            // Nothing to verify; just run the code.
            Console.Clear();
        }
    }
```

File: src/System.Console/tests/WindowAndCursorProps.cs (hunk 1 above)
Comment (hughbe) [reply src/System.Console/tests/WindowAndCursorProps.cs]: Fixed...

**Pull request merged**